packages/nulp_elite/src/pages/profile/Profile.js

```diff
@@ -62,14 +62,8 @@ const CssTextField = styled(TextField)({
 const Profile = () => {
   const { t } = useTranslation();
   const [userData, setUserData] = useState(null);
-  const [certData, setCertificateCountData] = useState({
-    totalCourses: 0,
-    certificatesReceived: 0,
-  });
-  const [courseData, setCourseCountData] = useState({
-    enrolledThisMonth: 0,
-    enrolledLastMonth: 0,
-  });
+  const [certData, setCertificateCountData] = useState({});
+  const [courseData, setCourseCountData] = useState({});
   const progressValue = 60; // Example value, you can set this dynamically based on your progress
```

Comment:
same here right?

Reply:
This is not used anywhere
progressValue